### PR TITLE
Installed bulma and created a custom scss file 

### PR DESCRIPTION
**Pull request by LexLeach**
##  Related Issues (include the `#`)
#45 

## Description of Changes Made
- Installed Bulma
- Installed Node-Sass
- Created custom scss file for dev-vs pallet to overwrite bulma defaults
- Complied modified css file with node-sass 
- added scripts to run node-sass and compile changes made within mystyles.scss

## Questions or Comments (optional)

For usage, see [Bulma Documentation](https://bulma.io/documentation/)
Our [wiki](https://github.com/SEI-39/developerverse/wiki) will be updated shortly with information as well

### CODE REVIEW

File: front-end/public/index.html [hunk 1]
```diff
@@ -2,42 +2,12 @@
...
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="stylesheet" href="../css/mystyles.css" />
     <title>React App</title>
```
Comment (midastouchprd) [inline front-end/public/index.html]: change to Developerverse

File: front-end/public/index.html [hunk 2]
```diff
@@ -2,42 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
```
Comment (midastouchprd) [inline front-end/public/index.html]: You should keep this meta tag

**Review by midastouchprd — CHANGES_REQUESTED** [review]
just some small stuff!

**Comment by LexLeach** [comment]
Updated

**Review by midastouchprd — APPROVED** [review]
LGTM

**Pull request merged**